### PR TITLE
replace the script action link

### PR DESCRIPTION
**Pull request by mumian**
### Best Practice Checklist

Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 
1. uri's compatible with all clouds (Stack, China, Government)
2. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
3. Use resourceGroup().location for resource locations
4. Folder names for artifacts (nestedtemplates, scripts, DSC)
5. Use literal values for apiVersion (no variables)
6. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
7. $schema and other uris use https
8. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
9. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checkist.md
- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.
### Changelog

*
*
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@mumian__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jgao). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: 101-hdinsight-linux-with-edge-node/azuredeploy.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
       "metadata": {
         "description": "A script action you can run on the empty node to install or configure additiona software."
       },
-      "defaultValue": "https://raw.githubusercontent.com/hdinsight/Iaas-Applications/master/EmptyNode/scripts/EmptyNodeSetup.sh"
+      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-hdinsight-linux-with-edge-node/scripts/EmptyNodeSetup.sh"
```
Comment (bmoore-msft) [inline 101-hdinsight-linux-with-edge-node/azuredeploy.json]: see the bp checklist for using artifacts in samples...

**Comment by bmoore-msft** [comment]
@mumian - are you able to deploy this successfully?  I can't see any reason why CI should fail but it did...

**Comment by mumian** [comment]
I used a List for URI, which shall be just a string.

**Pull request merged**